shopyo/modules/contact/templates/contact/contact_form.html

```diff
@@ -11,6 +11,9 @@
 
 {% block content %}
 
+<a href="{{url_for('contact.dashboard')}}"><button class="btn btn-info">view contact dashboard</button></a>
+<br><br>
+
```

Comment:
Contact form should be public facing. Normal users should not see the dashboard button. Maybe add {% if current_user.is_admin ...